Marlin/src/lcd/language/language_en.h

```diff
@@ -78,6 +78,10 @@ namespace Language_en {
   LSTR MSG_AUTO_HOME_I                    = _UxGT("Home ") LCD_STR_I;
   LSTR MSG_AUTO_HOME_J                    = _UxGT("Home ") LCD_STR_J;
   LSTR MSG_AUTO_HOME_K                    = _UxGT("Home ") LCD_STR_K;
+  LSTR MSG_AUTO_HOME_M                    = _UxGT("Home ") LCD_STR_M;
+  LSTR MSG_AUTO_HOME_O                    = _UxGT("Home ") LCD_STR_O;
+  LSTR MSG_AUTO_HOME_P                    = _UxGT("Home ") LCD_STR_P;
+  LSTR MSG_AUTO_HOME_Q                    = _UxGT("Home ") LCD_STR_Q;
```

Comment:
I think strings that contain axis names will need to get the `lcd_put_u8str_ind_P` treatment, so the `@` character will stand in for the axis name, similar to how `=`, `~`, and `*` are now used as placeholders for 1-based index, 0-based index, and indexed-E. That will reduce each set of 10 strings down to just one string.